hexrd/transforms/cpp_sublibrary/src/inverse_distortion.cpp

```diff
@@ -0,0 +1,29 @@
+#include <pybind11/pybind11.h>
+#include <pybind11/eigen.h>
+#include <Eigen/Core>
+#include <xsimd/xsimd.hpp>
+
+namespace py = pybind11;
+constexpr double FOUR_THIRDS_PI = 4.1887902;
+constexpr double N_THREE_HALVES_SQRT_3 = -2.59807621;
+constexpr double TWO_OVER_SQRT_THREE = 1.154700538;
+
+Eigen::ArrayXXd ge_41rt_inverse_distortion(const Eigen::ArrayXXd& inputs, const double rhoMax, const Eigen::ArrayXd& params) {
```

Comment:
I believe that in some cases, we can avoid copying the arrays when going back and forth between numpy and Eigen. We should try to do that when possible. See [here](https://pybind11.readthedocs.io/en/stable/advanced/cast/eigen.html#eigen). We might want to consider passing arguments like this instead:
```
Eigen::Ref<Eigen::ArrayXXd> inputs
```

Reply:
[Here](https://pypi.org/project/eigenpy/) is an interesting looking library that also claims to avoid copying when converting between numpy and eigen.